vmaas/reposcan/reposcan.py

```diff
@@ -749,6 +764,39 @@ def run_task(*args, **kwargs):
         return "OK"
 
 
+class ReleaseSyncHandler(SyncHandler):
+    """Handler for RHEL OS release version sync API."""
+
+    task_type = "Sync RHEL releases"
+
+    @classmethod
+    def put(cls, **kwargs):
+        """Sync RHEL releases."""
+        status_code, status_msg = cls.start_task()
+        return status_msg, status_code
+
+    @staticmethod
+    def run_task(*args, **kwargs):
+        """Function to start syncing RHEL releases."""
+        try:
+            init_logging()
+            init_db()
+            _, _, releases = SyncHandler.fetch_git()
+            controller = ReleaseController()
+            controller.store(releases)
+        except Exception as err:  # pylint: disable=broad-except
+            msg = "Internal server error <%s>" % hash(err)
+            LOGGER.exception(msg)
+            RELEASE_FAILED_IMPORT.inc()
```

Comment:
could be incremented twice if it fails in `controller.store(releases)`, shouldn't it?

Reply:
yes, but the counter is only for error indication, doesn't need to be precise